### DISCUSSION TITLE
I wrote an introduction to scraping with freshrss in German

### DISCUSSION THREAD

**Discussion by ulrischa** [FreshRSS/FreshRSS#5984] [Show and tell]
If somebody is interested and speaks German, here is my introduction in German:
https://www.ulrischa.de/web-scraping-mit-freshrss/

**Comment by Alkarex** [comment]
Thanks @ulrischa 👍🏻 
Maybe @math-GH could find a place for it in our documentation. Otherwise it is fine here in the *Show and tell* section :-)

**Reply by math-GH** [reply]
Thanks @ulrischa . The link is added to our article list now: https://github.com/FreshRSS/freshrss.org/issues/9#issuecomment-1870582285
